infrastructure/metadata/infrastructure/webdriver/bidi/set_permission.https.html.ini

```diff
@@ -0,0 +1 @@
+disabled: https://github.com/web-platform-tests/wpt/issues/47544
```

Comment:
Looks like there is a test failure for this test https://dev.azure.com/web-platform-tests/wpt/_build/results?buildId=130038&view=logs&jobId=af75b4de-b26f-5d4d-58e8-b41258076d44&j=af75b4de-b26f-5d4d-58e8-b41258076d44&t=2e0f365d-858c-52d3-7da5-72c4b225c9fa so perhaps it does not get disabled properly

Reply:
right, I messed up with the pas. Fixed now.